PopupTabLayout/tests/PopupTabLayout-specs.js

```diff
@@ -72,4 +73,140 @@ describe('PopupTabLayout specs', () => {
 			expect(handleClose).not.toHaveBeenCalled();
 		});
 	});
+
+	test('should display items from second tab', () => {
+		render(
+			<FloatingLayerController>
+				<PopupTabLayout open>
+					<Tab title="First Tab Title">
+						<TabPanels>
+							<TabPanel>
+								<Item>Item 1</Item>
+								<Item>Item 2</Item>
+							</TabPanel>
+						</TabPanels>
+					</Tab>
+					<Tab title="Second Tab Title">
+						<TabPanels>
+							<TabPanel>
+								<Item>Item 3</Item>
+								<Item>Item 4</Item>
+							</TabPanel>
+						</TabPanels>
+					</Tab>
+				</PopupTabLayout>
+			</FloatingLayerController>
+		);
+
+		let thirdItem = screen.queryByText('Item 3');
+		let fourthItem = screen.queryByText('Item 4');
+
+		expect(thirdItem).not.toBeInTheDocument();
+		expect(fourthItem).not.toBeInTheDocument();
+
+		const secondTab = screen.getByText('Second Tab Title');
+
+		userEvent.click(secondTab);
+
+		thirdItem = screen.getByText('Item 3');
+		fourthItem = screen.getByText('Item 4');
+
+		expect(thirdItem).toBeInTheDocument();
+		expect(fourthItem).toBeInTheDocument();
+	});
+
+	test('should fire onTabClick event for 5-way Select on tab element', () => {
+		const onTabClick = jest.fn();
+		render(
+			<FloatingLayerController>
+				<PopupTabLayout open>
+					<Tab onTabClick={onTabClick} title="First Tab Title">
+						<TabPanels>
+							<TabPanel>
+								<Item>Item 1</Item>
+								<Item>Item 2</Item>
+							</TabPanel>
+						</TabPanels>
+					</Tab>
+					<Tab title="Second Tab Title">
+						<TabPanels>
+							<TabPanel>
+								<Item>Item 3</Item>
+								<Item>Item 4</Item>
+							</TabPanel>
+						</TabPanels>
+					</Tab>
+				</PopupTabLayout>
+			</FloatingLayerController>
+		);
+
+		const firstTab = screen.getByText('First Tab Title');
+
+		fireEvent.keyDown(firstTab, {keyCode: 13});
+		fireEvent.keyUp(firstTab, {keyCode: 13});
```

Comment:
I've already used this way many times, but it's hard to know what key number 13 means.
I think the Slider-spec is more readable.

```

const keyDown = (keyCode) => (elm) => fireEvent.keyDown(elm, {keyCode});

const leftKeyDown = keyDown(37);
const rightKeyDown = keyDown(39);
const upKeyDown = keyDown(38);
const downKeyDown = keyDown(40);

```

CC: @daniel-stoian-lgp @adrian-cocoara-lgp @alexandrumorariu @andreitirlalge